webpack.config.js

```diff
@@ -95,7 +100,27 @@ module.exports = {
       },
       {
         test: /\.css$/,
-        use: ["style-loader", "css-loader"]
+        include: [
```

Comment:
It's better that each component from PF/react-core will import what css files it needs instead of importing everything by ourselves.
Or what are you actually trying to do here?